config/dev.exs

```diff
@@ -0,0 +1,5 @@
+use Mix.Config
+
+config :stump, time_api: Stump.Time.DateTime
```

Comment:
why not in config.exs?

Reply:
So I have that in config.exs but when I tried to use it in Fetcher it didn't pick up the DateTime function and returns `nil` for the function instead. This is the only thing I could think of that I would be missing unless we want to put:
`config :stump, time_api: Stump.Time.DateTime`
In the config of every project, which personally I don't think is a good idea.

Reply:
Should we put config under a keyword of stump? I am wondering if we should use logger or logger_wrapper as a generic word to describe what it means. Maybe stump does this, but it seems like we are tying our config to a sepcific library. I know we have this, but we wouldn't namespace sinatra or httprb if we were specifying config.

Reply:
This is the stump library, so it makes sense for the configs to be name spaced for stump right? And if you want to change in the config for stump in your application, it is of course going to be tied to stump, different libraries have different configurations.  

Should all of ingress' config also be name spaced as `:web_thing` in case we change the name ingress to something else? 😛

Reply:
If it's stump specific then it makes sense. I see the value is a stump date time so I agree with you. I thought this was the Ingress repo, but I see that was a different PR.